guides/common/modules/proc_installing-convert2rhel.adoc

```diff
@@ -0,0 +1,27 @@
+[id="installing-convert2rhel_{context}"]
+
+= Installing Convert2RHEL
+
+The following procedure makes Convert2RHEL available to your system.
```

Comment:
This sentence is odd. Can you reword it please?

Reply:
Changed.